src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-solution-proposal-step.html

```diff
@@ -0,0 +1,389 @@
+<!-- PROBLEM SOLVED -->
+<div data-ng-if="$ctrl.currentLineDiagnostic.data.answers.resolvedAfterTests">
+    <img class="col-md-2" src="assets/images/diagnostic/human-good.svg" width="180" height="180" />
```

Comment:
alt text

Reply:
also, not responsive friendly, CX should provide you with images of different sizes or the units should be percentages